tox.ini

```diff
@@ -1,39 +1,34 @@
 [tox]
-envlist = py27,py36,static,pidiff
+envlist = py27, py36
 
 [testenv]
-deps=-rtest-requirements.txt
-commands=pytest -v {posargs}
-whitelist_externals=sh
+deps =
+	# Note: we need to explicitly list requirements.txt here
+	# so it's processed at the same time as the constraints file
+	-rrequirements.txt
+	-rtest-requirements.txt
+commands = pytest {posargs}
+whitelist_externals = sh
 
-[testenv:py27]
-deps=
-  # Note: we need to explicitly list requirements.txt here
-  # so it's processed at the same time as the constraints file
-  -rrequirements.txt
-  -rtest-requirements.txt
-
-[testenv:static]
-deps=
-  pylint
-commands=
-  sh -c 'pylint-2 alt-src; test $(( $? & (1|2|4|32) )) = 0'
+[testenv:py26]
+# Defines the py26 env for Travis, as it isn't included with tox
 
 [testenv:cov]
-deps=
-  -rtest-requirements.txt
-  pytest-cov
-usedevelop=true
-commands=
-  pytest --cov-report=html --cov=pubtools {posargs}
+basepython = python2.7
+commands = pytest --cov-report=html --cov=. {posargs}
 
 [testenv:cov-travis]
 passenv = TRAVIS TRAVIS_*
-deps=
-  -rtest-requirements.txt
-  pytest-cov
-  coveralls
-usedevelop=true
-commands=
-  pytest --cov=pubtools {posargs}
-  coveralls
+basepython = python2.7
+deps =
+	-rrequirements.txt
+	-rtest-requirements.txt
+	coveralls
+usedevelop = true
+commands =
+	pytest --cov=. {posargs}
```

Comment:
cov=., means that it would also gather coverage data for the tests themselves, any example scripts added to the repo and so on.  It could be a problem later on because tests often end up with some code which isn't meant to be reached (e.g. code which is only executed if the tests are failing).   I'm pointing this out now but I think it's fine to handle those problems if/when they occur.

Reply:
Yeah, once alt-src is packaged correctly, we can direct cov to that dir.